### DISCUSSION TITLE
CDN 一栏更新后怎么感觉变糊了？是不是 DPI 缩放出问题了

### DISCUSSION THREAD

**Discussion by TPAXcc** [ywmoyue/biliuwp-lite#273] [General]
![图片](https://github.com/ywmoyue/biliuwp-lite/assets/91980905/bf687808-de63-433e-850c-872125d742ef)
只有展开后有这个问题，不展开是正常的
![图片](https://github.com/ywmoyue/biliuwp-lite/assets/91980905/43ed8176-3ef1-4d91-8be6-06fc4448927f)

**Comment by TPAXcc** [comment]
另外延迟建议右对齐然后加册加上层遮罩
类似这样
![图片](https://github.com/ywmoyue/biliuwp-lite/assets/91980905/54dd4420-323e-450d-8ad7-712202cae682)

**Comment by ywmoyue** [comment]
可能我显示器不行？没感觉出有变糊

**Comment by TPAXcc** [comment]
分辨率：2560x1600
DPI 缩放：124%
应该...不是你显示器的问题
![图片](https://github.com/ywmoyue/biliuwp-lite/assets/91980905/927b5e07-7a3d-49d5-a807-a1ef5d37fbdb)
注意看上边跟下边的字体边缘

**Comment by ywmoyue** [comment]
我看你截图是挺明显的，但我自己这边启动应用看并不明显

**Comment by TPAXcc** [comment]
> 我看你截图是挺明显的，但我自己这边启动应用看并不明显

那估计就是分辨率跟dpi不一样了，我这里是32寸的，没准跟尺寸也有关系？

**Comment by TPAXcc** [comment]
![图片](https://github.com/ywmoyue/biliuwp-lite/assets/91980905/866cf0d5-8f3d-44a0-b48f-eb9519360bb3)
测试 DPI 124% 打开模糊修复

测试 DPI 124% 关闭模糊修复
![图片](https://github.com/ywmoyue/biliuwp-lite/assets/91980905/1970980d-3204-4222-a5f6-4883cb0a452e)

测试 DPI 100% 打开模糊修复
![图片](https://github.com/ywmoyue/biliuwp-lite/assets/91980905/a8682103-8970-47e2-bd23-0ca52c987ff4)

**Comment by ywmoyue** [comment]
可能WinUI的问题，我也没法解决，由于不影响使用，此issue移动至Discussion
